tests/all_providers/config.toml

```diff
@@ -1,5 +1,5 @@
 [core_settings]
-log_level = "debug"
+log_level = "error"
```

Comment:
I had to do this to limit the log size, otherwise the build was failing on Travis CI.